sretoolbox/container/image.py

```diff
@@ -57,14 +57,31 @@ class Image:
     :param password: (optional) The private registry password
     :param auth_server: (optional) The host that the username and password are
                         meant for
+
+    :param response_cache (optional) A cache to store responses,
+    indexed by both etag and digest
     """
+
+    ACCEPT_HEADERS = {
+        'Accept':
+        'application/vnd.docker.distribution.manifest.v1+json,'
+        'application/vnd.docker.distribution.manifest.v2+json,'
+        'application/vnd.docker.distribution.manifest.v1+prettyjws,'
+    }
+
+    MAX_CACHE_ITEM_SIZE = 50*1024
```

Comment:
It is a class variable, it will only be computed when the class is first loaded. 50KB is an arbitrary number - most manifests I'm seeing are on the order of 10-15KB, so 50KB gives us enough room for larger instances.

I don't want to store arbitrarily large items in the cache, to protect us from OOM-ing too easily.

Reply:
Awesome, please capture that info a comment